tests/libdv/dv.go

```diff
@@ -156,8 +178,7 @@ func randName() string {
 }
 
 // PVC Options
-
-// PVCWithStorageClass add the sc storage class name to the DV
+// StorageWithStorageClass add the sc storage class name to the DV
```

Comment:
comment changed here for the wrong function

Reply:
Fixed